src/Compilers/CSharp/Portable/Syntax/InternalSyntax/SyntaxListBuilder.cs

```diff
@@ -2,17 +2,18 @@
 
 using System;
 using System.Diagnostics;
+using Microsoft.CodeAnalysis.Syntax.InternalSyntax;
 
 namespace Microsoft.CodeAnalysis.CSharp.Syntax.InternalSyntax
 {
-    internal class SyntaxListBuilder
+    internal class SyntaxListBuilder : AbstractSyntaxListBuilder
     {
-        private ArrayElement<CSharpSyntaxNode>[] _nodes;
+        private ArrayElement<CSharpSyntaxNode>[] Nodes;
```

Comment:
Our coding guidelines are that private instance fields should be named like `_nodes`.